tests/websocket.rs

```diff
@@ -23,3 +25,37 @@ async fn connect_app_websocket() {
 
     assert_eq!(apps.len(), 2);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn connect_admin_websocket() {
+    let mut sweet_conductor = SweetConductor::from_standard_config().await;
+
+    let admin_port = sweet_conductor
+        .get_arbitrary_admin_websocket_port()
+        .unwrap();
+    let url = format!("localhost:{admin_port}");
```

Comment:
Not a valid URL, doesn't have a scheme, this is why I'm asking for this to be something other than a string above.